types/stripe/index.d.ts

```diff
@@ -5206,7 +5208,1262 @@ declare namespace Stripe {
+				 */
+				approved: boolean;
+
+				/**
+				 * How the card details were provided. One of chip, contactless, keyed_in, online, or swipe.
+				 */
+				authorization_method: AuthorizationMethod;
+
+				/**
+				 * The amount that has been authorized. This will be 0 when the object is created, and increase after it has been approved.
+				 */
+				authorized_amount: number;
+
+				/**
+				 * The currency that was presented to the cardholder for the authorization. Three-letter ISO currency code, in lowercase. Must be a supported currency.
+				 */
+				authorized_currency: string;
+
+				balance_transactions: balance.IBalanceTransaction[];
+
+				/**
+				 * Show child attributes
+				 */
+				card: cards.IIssuingCard;
+
+				/**
+				 * The cardholder to whom this authorization belongs.
+				 */
+				cardholder: string | cardholders.ICardholder;
+
+				/**
+				 * Time at which the object was created. Measured in seconds since the Unix epoch.
+				 */
+				created: number;
+
+				/**
+				 * The amount the authorization is expected to be in held_currency. When Stripe holds funds from you, this is the amount reserved for the authorization. This will be 0 when the object is created, and increase after it has been approved. For multi-currency transactions, held_amount can be used to determine the expected exchange rate.
+				 */
+				held_amount: number;
+
+				/**
+				 * The currency of the held amount. This will always be the card currency.
+				 */
+				held_currency: string;
+
+				is_held_amount_controllable: boolean;
+
+				/**
+				 * Has the value true if the object exists in live mode or the value false if the object exists in test mode.
+				 */
+				livemode: boolean;
+
+				merchant_data: MerchantData;
+
+				/**
+				 * Set of key-value pairs that you can attach to an object. This can be useful for storing additional information about the object in a structured format.
+				 */
+				metadata: IMetadata;
+
+				/**
+				 * The amount the user is requesting to be authorized. This field will only be non-zero during an issuing.authorization.request webhook.
+				 */
+				pending_authorized_amount: number;
+
+				/**
+				 * The additional amount Stripe will hold if the authorization is approved. This field will only be non-zero during an issuing.authorization.request webhook.
+				 */
+				pending_held_amount: number;
+
+				/**
+				 * Show child attributes
+				 */
+				request_history: RequestHistory[];
+
+				/**
+				 * One of closed, pending, or reversed.
+				 */
+				status: AuthorizationStatus;
+
+				/**
+				 * Show child attributes
+				 */
+				transactions: transactions.ITransaction[];
+
+				/**
+				 * Show child attributes
+				 */
+				verification_data: VerificationData;
+
+				/**
+				 * What, if any, digital wallet was used for this authorization. One of apple_pay, google_pay, or samsung_pay.
+				 */
+				wallet_provider: WalletProvider;
+			}
+			interface MerchantData {
```

Comment:
There should be an empty newline between interfaces.

Reply:
@pushplay resolved. Let me know how it looks now.